core/src/transport.cpp

```diff
@@ -134,11 +134,13 @@ namespace irods::http
 	{
 		boost::asio::ssl::context ctx{boost::asio::ssl::context::tlsv12_client};
 		const auto config{irods::http::globals::oidc_configuration()};
+		ctx.set_default_verify_paths();
 
-		const std::string& cert_path{config.at("tls_certificates_directory").get_ref<const std::string&>()};
-		ctx.add_verify_path(cert_path);
-		ctx.set_verify_mode(boost::asio::ssl::verify_peer);
+		if (const auto cert_path{config.find("tls_certificates_directory")}; cert_path != std::end(config)) {
+			ctx.add_verify_path(cert_path->get_ref<const std::string&>());
```

Comment:
Does this ADD to the set of paths to search or override it?

Reply:
i hope add.    add is what we want i think.

Reply:
Seems pretty easy to test, in theory. Put certificates in the default location, specify an empty `tls_certificates_directory`, see if it works?

Reply:
Adding to what @alanking said - can also move a self-signed cert between the two locations to see how the HTTP API behaves.

Reply:
> We could test a self-signed cert, but I would rather move all the default certs to a separate directory, leaving the default cert directory empty, and point `tls_certificates_directory` to the new directory to see that everything still works fine.

Are you talking about moving the system certs from their normal location?